doc/user/content/sql/select.md

```diff
@@ -149,14 +153,14 @@ columns. If an unqualified name refers to both an input and output column,
 
 ## Examples
 
-### Creating a view
+### Creating a materialized view
```

Comment:
ISTM like we want to encourage creating an indexed view, rather than a materialized view here?

Reply:
Ok, I changed the example to be an indexed view.